### PR TITLE
add ALL the qualifications

### PR DESCRIPTION
**Pull request by Anbranin**
Hopefully this is somewhat clear

### CODE REVIEW

**Comment by werebus** [comment]
I've spotted a possible use for labels, though: Those (optional) issues, perhaps?

File: config/qualifications.yml [hunk 1]
```diff
@@ -1,35 +1,39 @@
...
-  Ask a supervisor to invite you to our Slack team and add you to the relevant channels.
+  We use [Slack](https://slack.com/) for communication every day. After your invitation to our
+  Slack team, you can download the desktop version or use the web version, whichever suits you.
 ---
-title: 'Complete a Rails Tutorial'
+title: Choose a text editor
+description: >
+  We don't care what text editor you use. However, we do have some suggestions.
+  Other people in the office have used [atom](https://atom.io/), [sublime](https://www.sublimetext.com/3)...
+  and [Vim](https://www.tutorialspoint.com/vim/vim_introduction.htm). Vim is already installed on your laptop,
```
Comment (jforgue) [inline config/qualifications.yml]: What about [Ruby Mine](https://www.jetbrains.com/ruby/)!?

File: config/qualifications.yml [hunk 2]
```diff
@@ -40,20 +44,70 @@ description: >
   - In section 1.5.1, there are instructions to set up a Heroku account to deploy your code to.
     Since you'll be deploying code to production in a few short months, you can skip all related steps.
 ---
-title: 'Complete a ruby tutorial'
+title: Complete a ruby tutorial
```
Comment (jforgue) [inline config/qualifications.yml]: Shouldn't ruby come before rails?

File: config/qualifications.yml [hunk 3]
```diff
@@ -40,20 +44,70 @@ description: >
...
+---
+title: Complete a git tutorial (optional)
 description: >
   If you can push, pull, merge, and resolve conflics with git, you're all set.
   Otherwise, consider a tutorial [like this](https://www.learnenough.com/git-tutorial/getting_started)
-title: 'Complete a command line tutorial (optional)'
+---
+title: Complete a command line tutorial (optional)
 description: >
   Having a good grasp of the command line is essential for working in our team,
```
Comment (jforgue) [inline config/qualifications.yml]: If it's essential why is it optional?

File: config/qualifications.yml (hunk 2 above)
Comment (Anbranin) [reply config/qualifications.yml]: "That's debatable" Matt said, and I agree. You don't have to know Ruby to learn Rails, in fact, and I think becoming proficient at Ruby first could be confusing. 

That being said, it doesn't really make sense to try to do most of this file in order, just the first couple bits. So in reality they can do whatever.

File: config/qualifications.yml (hunk 3 above)
Comment (Anbranin) [reply config/qualifications.yml]: If someone already has a good grasp of the command line, they don't have to complete a tutorial. Maybe I can state somewhere "if you can already do this, just check it off" but that's kind of true of most of the items.

**Pull request merged**